flocker/control/_model.py

```diff
@@ -804,6 +804,8 @@ class IClusterStateSource(Interface):
     This is presently used for activity/inactivity tracking to inform change
     wiping.
     """
+    # XXX Should this interface be expanded to include the node UUID and the
+    # type of agent (container or dataset) that reported the state change?
```

Comment:
:question: so yeah, looking through the code that uses the implementation of `IClusterStateSource` I can see *where* we use it, but I'm not clear on what it's doing that's actually useful. It seems we should either not bother logging a `STATE_SOURCE` field, or an implementation of `IClusterStateSource` that reports specific information about the source (UUID, agent, etc.) should be used. Or possibly this information is already represented somewhere in the log tree by the time we get to logging these causes of `send_cluster_state` and when the logs are correctly parsed, there's no benefit to additional source logging here.